lesson02/elb_asg/common_vars.tf

```diff
@@ -0,0 +1,28 @@
+variable "aws_region" {
+  default = "us-east-1"
+}
+
+variable "metadata" {
+  default = {
+    appname    = "sample-app"
+    appversion = "latest"
+  }
+}
+
+variable "env" {
+  description = "string for the enviroment, allowed values develop, uat, prod"
```

Comment:
can we use complete names instead: development, uat, production?

Reply:
per aws resouce name limitation we should use short names